src/lwc/geFormWidgetAllocation/geFormWidgetAllocation.html

```diff
@@ -1,15 +1,12 @@
 <template>
-    <lightning-layout if:true={hasAlert}>
-        <lightning-layout-item size="12">
-            <div class={alertClass} role="alert">
-                <span class="slds-assistive-text">Allocation error</span>
-                <span class="slds-icon_container slds-icon-utility-error slds-m-right_x-small">
-                    <lightning-icon icon-name={alertIcon} variant="inverse" size="x-small"></lightning-icon>
-                </span>
-                <h2>{alertBanner.message}</h2>
-            </div>
-        </lightning-layout-item>
-    </lightning-layout>
+    <c-alert-banner if:true={hasAlert}
+            theme={alertClass}
+            assist-text="Allocation Error"
```

Comment:
Could we use a custom label here?

Reply:
+1 to custom label

Reply:
Is this a unit test or a customer-facing error?

Reply:
@melissabarber this would be a message used with accessibility

Reply:
K, I need a little extra help understanding what the error needs to convey.

Reply:
@melissabarber this would just be conveying that where's an error that has occurred in the GAU widget, but not specifically what the error is.

Reply:
@dfuller114 should the error be specific to the GAU Allocations field bundle or should we use a variable that could pull in the name of any of the field bundles?

Reply:
@melissabarber since it's referenced in the GAU widget, this would only be specific to a GAU widget error

Reply:
@dfuller114  Totally get that, I'm wondering if we create a custom label for this should the custom label contain a variable to pull in the name of the field bundle so the custom label could be reused for other purposes.

Reply:
Chatted more with @ptewson-sfdo and I recommend making this custom label read, "GAU Allocation Error".

Reply:
added the label!